pkg/controller/migplan/pvlist.go

```diff
@@ -409,3 +420,15 @@ func isRWO(accessModes []core.PersistentVolumeAccessMode) bool {
 	}
 	return false
 }
+
+func pvcInPodVolumes(pvc core.PersistentVolumeClaim, pods []core.Pod) bool {
+	for _, pod := range pods {
+		for _, volume := range pod.Spec.Volumes {
+			if volume.PersistentVolumeClaim != nil && volume.PersistentVolumeClaim.ClaimName == pvc.Name {
```

Comment:
We also need to compare pvc namespace and pod namespace. If pvc and pod are in different namespace, we can skip the inner loop entirely.

Reply:
Also, we're now looking at pods. Do we also need to look at templates if it's not found in the pods? Wondering whether skip should be a valid option for PVCs that belong to scaled-down DeploymentConfigs. @dymurray what do you think here?